content/2021/08/simplex-uses-minfraud-data.mdx

```diff
@@ -1,29 +1,64 @@
+### “The more data I have, the more significant my results can be. More data
+means that we can do better research.”
 
-The variety of data available through minFraud Insights allows Simplex to track patterns across many different data points. Rather than trying to anticipate which data points will be most relevant to fraud, Simplex can feed the breadth of minFraud data into their algorithm, allowing it to identify and adapt to patterns in fraud as they emerge. This adaptability is key to fighting fraud in the rapidly changing crypto industry.
+The variety of data available through minFraud Insights allows Simplex to track
+patterns across many different data points. Rather than trying to anticipate
+which data points will be most relevant to fraud, Simplex can feed the breadth
+of minFraud data into their algorithm, allowing it to identify and adapt to
+patterns in fraud as they emerge. This adaptability is key to fighting fraud in
+the rapidly changing crypto industry.
 
-### “We also make minFraud data available to our fraud analysts for manual review of transactions.”
+### “We also make minFraud data available to our fraud analysts for manual
+review of transactions.”
 
-The data from minFraud Insights also helps Simplex to dig into particular claims of fraud, allowing their fraud prevention specialists to avoid erroneous chargebacks and investigate issues thoroughly when they arise. When a customer claims that a transaction is fraudulent, Simplex fraud investigators can check the details of their customer’s story against the data returned by minFraud, to confirm their story or ask more questions if the data and the story don’t seem to line up.
+The data from minFraud Insights also helps Simplex to dig into particular claims
+of fraud, allowing their fraud prevention specialists to avoid erroneous
+chargebacks and investigate issues thoroughly when they arise. When a customer
+claims that a transaction is fraudulent, Simplex fraud investigators can check
+the details of their customer’s story against the data returned by minFraud, to
+confirm their story or ask more questions if the data and the story don’t seem
+to line up.
 
-Along with a powerful algorithm, Simplex’s analysts use minFraud Insights data to protect the integrity of their clients’ transactions and reduce costs due to chargebacks. In the end, whether they’re building a fraud detection algorithm or manually reviewing transactions, it all comes down to the data.
+Along with a powerful algorithm, Simplex’s analysts use minFraud Insights data
+to protect the integrity of their clients’ transactions and reduce costs due to
+chargebacks. In the end, whether they’re building a fraud detection algorithm or
+manually reviewing transactions, it all comes down to the data.
 
-### “The quality of the data is really important to us. That’s why we use minFraud.”
+### “The quality of the data is really important to us. That’s why we use
```

Comment:
I think this should be a block quote rather than a header. However, if it is preferred to stay a header, only part of the quote is currently a heading.